appgate/__main__.py

```diff
@@ -59,9 +59,12 @@ async def dump_entities(ctx: Context, output_dir: Optional[Path],
 
 def main_dump_entities(args: OperatorArguments,  stdout: bool = False,
                        output_dir: Optional[Path] = None) -> None:
-    asyncio.run(dump_entities(ctx=get_context(args),
-                              output_dir=output_dir,
-                              stdout=stdout))
+    try:
+        asyncio.run(dump_entities(ctx=get_context(args),
+                                  output_dir=output_dir,
+                                  stdout=stdout))
+    except AppgateException:
+        log.error('[dump-entities] Unable to perform operation')
```

Comment:
Don't we want to return some kind of top level error?
E.g. log the error and set a non-zero exit code.

Reply:
ahh yes missing exit(1)

Reply:
```oot@337032a6b774:/project# appgate-operator --spec-directory api_specs/v13 dump-entities --directory example-v13-entities 2> /dev/null || echo "OHHHHHHHHHH so bad"
OHHHHHHHHHH so bad
```